deploy/eck-integrations/values.yaml

```diff
@@ -0,0 +1,93 @@
+
+
+elasticsearch:
+  host: ""
+  user: ""
+  pass: ""
+
+cloudDefend:
+  enabled: false
+
+kubernetes: 
```

Comment:
Why is kubernetes a top level refernece here? If we refer to the package then I think we need to introduce the integration/ pckage as top and then kubernetes

Reply:
We can't create top level references for specific integrations. If cloudDefend is the integration that we are going to install then this value should only surface on the corresponding values.yaml same for Kubernetes being a top-level reference

Reply:
I think that this was already covered in one of our meetings, and as we discussed this helm chart is integration-focused thus the integrations are at the top level and then the helm chart templates transform all integrations and tight them to the specific agents. Please if you consider this as unresolved feel free to re-open 🙂